changedetectionio/content_fetchers/base.py

```diff
@@ -146,6 +146,11 @@ def iterate_browser_steps(self):
                     if '{%' in step['selector'] or '{{' in step['selector']:
                         selector = jinja_render(template_str=step['selector'])
 
+                    # If the operation is Goto site, change it to Goto URL and use the url as the optional value
+                    if step['operation'] == 'Goto site':
+                        step['operation'] = 'Goto URL'
+                        optional_value = url
```

Comment:
the function has `url` but its only used here? seems weird

Reply:
Done.